src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,47 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Book {
+
+    @Id
+    private long id;
+
+    private String genre;
```

Comment:
Wow! I thought that Column annotation was required each time you added a field, but it seems that you are right and it is not needed unless you modify default behaviour. Awesome! :1st_place_medal: